### ISSUE TITLE
v2 of Va Facility Selection page x Staging Review Accessibility Checklist

### ISSUE THREAD

**Issue by lenaecb** [department-of-veterans-affairs/va.gov-team#14169]
## Scope
- new v2 of VA facility selection page
- includes alert modals

## Acceptance Criteria
- Complete required accessibility checklist items for staging review:
  - [x] Read the [accessibility staging review documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#accessibility-staging-review-preparation)
  - [x] [Axe scans in daily development](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#axe-scans-in-daily-development)
    - Conduct axe browser scan, and document or address issues found
  - [x] [Axe scans in end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#axe-scans-in-end-to-end-tests)
    - Ensure end-to-end tests - including the axe check, pass
  - [x] [Color tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#color-tests)  
    - Check color pairing for proper contrast
    - Check for colorblindness issues
  - [x] [Content resize check](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#content-resize-check)
    - Zoom to 200% at 1280px wide and confirm readability, no info cut off, etc
    - Repeat for 300%
    - Repeat for 400%
  - [x] [Keyboard navigation check](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#keyboard-navigation-check)
    - Navigate all pages and core functionality with keyboard only
  - [x] [Screen reader tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#screen-reader-tests)
    - Test all pages and core functionality happy path with MacOs + VoiceOver and/or NVDA+Firefox
- [ ] Check off items as they are completed

## Reference
- [VSP Collaboration Cycle](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle-visual.pdf)
- [Accessibility staging review preparation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#accessibility-staging-review-preparation)

## How to configure this issue
- [ ] **Attach to Collab Cycle Epic**
- [ ] **Labeled with Team**

**Comment by jbalboni** [comment]
We may want to consider these guidelines as well, for the "show more locations" link: https://dsva.slack.com/archives/C8E985R32/p1601055663006900

We could probably skip the aria-live part until we get Trevor to look at the page.

**Comment by narin** [comment]
The Cypress tests I've created are passing locally but failing in Jenkins for some reason.  However the page passes the aXe tests using the chrome extension, So I think we can loop @1Copenut in for an accessibility audit as I figure out what's going on with the Cypress tests @lenaecb.
